readme.md

```diff
@@ -2,25 +2,81 @@
 
 Rainbow bridge for shipping your files to any cloud storage service with the same function calls.
 
-![image](https://user-images.githubusercontent.com/59074379/226159115-1cfcb221-127f-4574-87ed-b74b4b2c4591.png)
+<img src="https://user-images.githubusercontent.com/59074379/226159115-1cfcb221-127f-4574-87ed-b74b4b2c4591.png" width="1000" />
+
+# Table of contents
+
+- [Bifrost](#bifrost)
+- [Problem Statement](#problem-statement)
+  - [Google Cloud Storage(GCS)](#google-cloud-storagegcs)
+  - [Pinata](#pinata)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+- [Changelog](#changelog)
+- [Contributors](#contributors)
 
 # Problem Statement
+Many projects need to store files in the cloud, and different projects might use different cloud storage providers. Using different SDKs with different implementations for each provider can be tedious and time-consuming. Bifrost aims to simplify the process of working with multiple cloud storage providers by providing a consistent API for all of them. To better understand how Bifrost solves this problem, let's take a look at two separate code samples for GCS and Pinata using conventional means and how Bifrost eases the steps. 
```

Comment:
Please don't remove the Thor reference...that's where the project idea came from. 
I think we can have this as 
```go
/*
Many projects need to store files in the cloud and different projects might use different cloud storage providers or, sometimes, multiple cloud providers all at once. Using different SDKs with different implementations for each provider can be tedious and time-consuming. Bifrost aims to simplify the process of working with multiple cloud storage providers by providing a consistent API for all of them. 

To better understand how Bifrost solves this problem, let's take you on a ride with Thor as we compare two separate code samples (conventional - bifrost) for using both Google Cloud Storage and Pinata Cloud in a single project.
*/
```